curvesim/metrics/base.py

```diff
@@ -356,6 +351,22 @@ def get_market_price(self, base, quote, prices):
 pandas_coin_pair_attr = {DataFrame: "columns", Series: "index"}
 
 
+def get_coin_pairs(prices):
+    """
+    Returns the coin pairs available in the price data.
+    """
+    if isinstance(prices, DataFrame):
+        return prices.columns
+    if isinstance(prices, Series):
+        return prices.index
```

Comment:
Should make sure we return the same type for each option

Reply:
All return tuples now. Added an explicit test that the same data and type are returned regardless of input format in test_metrics.test_get_poin_pairs